src/ui/interface.js

```diff
@@ -1971,61 +1971,6 @@ export class UI {
 		$j('#playerinfo .units span').text(userTurn.getNbrOfCreatures() + ' / ' + game.creaLimitNbr);
 	}
 
-	// Broken and deprecated
-	showStatModifiers(stat) {
```

Comment:
None of this code was being used (assume it's related to the UI you said broke a while ago), so I've removed it to avoid confusion.